### ISSUE TITLE
Can't manage to make extension packs working

### ISSUE THREAD

**Issue by ikki-ai** [youngj/EnvayaSMS#42]
How to install extension pack?
When i install apk extension pack it Saïd it's done but i still can't send more than 100 SMS/hour...
Anybody knows how WE should install it ?

**Comment by fictus** [comment]
Yeah, I had the same issue - it didn't work even after installing the extension packs.

Good thing the source code is available here! Since sms limits are not an issue any more in the current Android versions I updated the app to default to 1000 sms an hour. took me a few hours to set up my local environment, but was finally able to build it and install it on my phone.

**Comment by ikki-ai** [comment]
Great IS it possible to share your apk ?
Thanks a lot

**Comment by fictus** [comment]
> Great IS it possible to share your apk ?
> Thanks a lot

Is there a way to PM in here?

**Comment by fictus** [comment]
Ok. Im going to post the link here, but WARNING: I did not test the app to see if it really handles 1000 sms in one hour! Would be interesting to have somebody send like 150 sms in one hour to test.

ALSO, I signed the app with a self-signed certificate which expires in August 2030 (maybe this modified app wont work anymore after that year?)

AND REALLY the best approach is to fix the issue with the expansion packs - something that I don't have the time to do (I'm not really a Java developer, I'm only experienced with C#).

With that understood, here's the link to the modified APK 
[HERE](https://drive.google.com/file/d/16B8GO_HVZKBgoZU31aEoLHFTpoywendZ/view?usp=sharing)
